javascript/chat.html

```diff
@@ -30,68 +30,17 @@
                         <input type="text" class="form-control message" placeholder="Your Message..." id="chat-input" onkeypress="checkSubmit(event)">
                         <span class="input-group-btn"><button class="btn btn-primary" type="submit" onclick="sendChat()">Send</button></span>
                     </div>
+                    <!-- <div class="row message">
```

Comment:
If this is just commented out can it be removed?

Reply:
All gone!